engine/access/ingestion/engine.go

```diff
@@ -360,14 +360,6 @@ func (e *Engine) processFinalizedBlock(block *flow.Block) error {
 		return fmt.Errorf("could not index block for collections: %w", err)
 	}
 
-	// loop through seals and index ID -> result ID
-	for _, seal := range block.Payload.Seals {
-		err := e.executionResults.Index(seal.BlockID, seal.ResultID)
```

Comment:
This is a redundant index, because sealed result can be queried by first get seal by block ID, and then get result by seal.ResultID.